### PR TITLE
Design fixes in multiple places

### PR DESCRIPTION
**Pull request by Mykhailo-Chudyk**
## Description

1. Aligned button design on the landing page with the branding kit.
2. Added "Business" to the list of majors.
3. Added `feedbackQuestion` to the "Explore More" mode.
4. Removed the question about the user address.

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests (adding or updating tests)
- [ ] Documentation update

## Testing Instructions

Follow the list from the description and verify that it works for you.

### CODE REVIEW

File: src/components/Card.tsx
Comment (avalucianelson) [inline src/components/Card.tsx:1]: As a potential improvement in the future, I think it'd make the most sense to have all four of these be different colors. That way, the different majors jump out at you on the homepage! :)

File: src/components/PersonalDetailsForm.tsx
Comment (avalucianelson) [inline src/components/PersonalDetailsForm.tsx:1]: Love that we implemented this change based on user feedback - tysm.

**Review by avalucianelson — COMMENTED** [review]
Thank you so much for this, Mkyhailo! This is truly a great PR. All of these things, even if they seem like quick switches, move the needle toward a more usable (and beautiful) product. The only potential improvement I could see would be adjusting so that Business has its own colored chip -- I think that would be more clear.

**Comment by ucey-star** [comment]
@Mykhailo-Chudyk please can you provide screenshots, just as a way to see the new feature/fix added

**Comment by Mykhailo-Chudyk** [comment]
@ucey-star I mentioned in the group chat that I can't see it locally, since I'm getting redirected to production. We need to fix that issue first before being able to see the changes

File: src/components/Card.tsx
Comment (Mykhailo-Chudyk) [reply src/components/Card.tsx:1]: Agreed!

**Comment by Mykhailo-Chudyk** [comment]
Here are the screenshots @ucey-star @avalucianelson. If that looks good to you, you can approve and merge this PR. If there are still some errors, please lmk, and I will update this PR tomorrow

image
image
image
image

**Comment by ucey-star** [comment]
First of all! Great PR Misha!

**Review by ucey-star — APPROVED** [review]
LGTM!

**Pull request merged**